adapters/MoPub/MoPubAdapter/MoPubAdapterMediatedNativeAd.m

```diff
@@ -128,6 +127,14 @@ - (void)mediatedNativeAd:(id<GADMediatedNativeAd>)mediatedNativeAd
   } else {
     MPLogWarn(@"Could not add impression trackers.");
   }
+    // Returning the main UIImageView via AdMob's GADMediaView
+    if ([view isKindOfClass:[GADNativeAppInstallAdView class]]) {
```

Comment:
Instead of placing the view here, the media view needs to be returned as an asset using the `GADMediatedNativeAppInstallAd`'s [`mediaView`](https://developers.google.com/admob/ios/api/reference/Protocols/GADMediatedNativeAppInstallAd#/c:objc(pl)GADMediatedNativeAppInstallAd(im)mediaView) method and the Google Mobile Ads SDK will take care of placing the view.

Please see the [Facebook adapter](https://github.com/googleads/googleads-mobile-ios-mediation/blob/bf4551e1824d267450923059c198cbaae92cfd17/adapters/Facebook/FacebookAdapter/GADFBNativeAd.m#L302) for a sample implementation.

Reply:
@sbagadi Thanks! MoPub does not provide a MediaView object that can be extracted and returned like FB does, so the adapter cannot make use of GADMediatedNativeAppInstallAd's mediaView method. What would you recommend? Is there any side effects to setting the MediaView where it currently is?

Reply:
If you are talking about when the native video would be added, I would need to take a look at the native video integration for MoPub works before I can recommend something.

But for now, the `UIImageView` that is being added here can be returned as the media view in the `mediaView` method.